zebra-chain/src/primitives/address.rs

```diff
@@ -2,12 +2,12 @@
 //!
 //! Usage: <https://docs.rs/zcash_address/0.2.0/zcash_address/trait.TryFromAddress.html#examples>
 
+use zcash_address::unified::{self, Container};
 use zcash_primitives::sapling;
 
-use crate::{orchard, parameters::Network, transparent, BoxError};
+use crate::{parameters::Network, transparent, BoxError};
 
 /// Zcash address variants
-// TODO: Add Sprout addresses
```

Comment:
Optional Nitpick: There's no sprout address implemented, so I'd like to leave this comment in.

sprout itself is out of scope because it's not used for coinbase transactions.